packages/web/src/pages/sign-on/components/desktop/ForgotPasswordHelper.tsx

```diff
@@ -46,6 +50,13 @@ export const ForgotPasswordHelper = ({
     copyToClipboard(messages.subject)
   }
 
+  const linkCss: CSSObject = {
```

Comment:
a little odd to me to use the "inline css api" for a stored abstraction instead of a class but nothing wrong with it 🤷

Reply:
yea apparently the css prop is more performant so trying to use it more

Reply:
Wait really? Interesting. I would have thought that a class would be marginally better here since it's build-time and (correct me if I'm wrong) but this here will end up creating two different classes (one for each Text)

Reply:
Either way its not a big deal, dont really need to modify for performance here no matter what 🤷